diff --git a/node_modules/electron-re/lib/libs/ProcessManager/index.js b/node_modules/electron-re/lib/libs/ProcessManager/index.js
index 4805121..cc2ffe2 100644
--- a/node_modules/electron-re/lib/libs/ProcessManager/index.js
+++ b/node_modules/electron-re/lib/libs/ProcessManager/index.js
@@ -113,7 +113,7 @@ var ProcessManager = /*#__PURE__*/function (_EventEmitter) {
         if (_this.pidList.length) {
           pidusage(_this.pidList, function (err, records) {
             if (err) {
-              console.log("ProcessManager: refreshList errored -> ".concat(err));
+              // console.log("ProcessManager: refreshList errored -> ".concat(err));
             } else {
               var _this$ui2;
 
